protobuf/model_config.proto

```diff
@@ -1578,6 +1584,70 @@ message ModelWarmup
   map<string, Input> inputs = 3;
 }
 
+//@@
```

Comment:
I would suggest nesting it inside ModelSequenceBatching. Just before message State. Follow how Control is described for ControlInput.

Reply:
Fixed